src/public.js

```diff
@@ -4,18 +4,25 @@
  */
 import isPublic from './helpers/isPublicPage.js'
 import isDocument from './helpers/isDocument.js'
+import isPdf from './helpers/isPdf.js'
```

Comment:
The NewFileMenu from files.js still seems relevant so maybe we move that over here as well. Should fix the cypress failure